### PR TITLE
rename

### LINKED ISSUE #486: cepp spelling
/cepp is spelled two different ways

Carole Epp
Carol Epp

We need to find the correct spelling and correct all the incorrect versions.

### PR DESCRIPTION
**Pull request by Aholicknight**
Carol Epp > Carole Epp

fixes #486

### CODE REVIEW

File: www/artists/cepp/artist.json [hunk 1]
```diff
@@ -4,5 +4,6 @@
   "changelog": [
     "Created by Grant Atter: 2010",
     "Edited by Rhiannon Riddell: 2017"
+	"Edited by Javin Zatwarniski in 2019"
```
Comment (synixebrett) [inline www/artists/cepp/artist.json]: Changelog is for content only, got history will handle fixes

File: www/artists/cepp/domesticwork.html [hunk 2]
```diff
@@ -58,12 +58,12 @@
...
   <img src="domestics/right_lean.png" class="c2" alt="no alt text">
   <p class="text">
-    These are the other pottery pieces of Carol's Work. Challenging uniformity and symmetry, Carol's sets are truly unique in how different they seem from one another.
+    These are the other pottery pieces of Carole's Work. Challenging uniformity and symmetry, Carole's sets are truly unique in how different they seem from one another.
   </p>
   <div id="content">
     <div id="footer">
       <div class="c3">
-        All images copyright of Carol Epp. Created by Grant Atter 2010. Edited by Rhiannon Riddell 2017.
+        All images copyright of Carole Epp. Created by Grant Atter 2010. Edited by Rhiannon Riddell 2017. Edited by Javin Zatwarniski in 2019.
```
Comment (synixebrett) [inline www/artists/cepp/domesticwork.html]: bruh

File: www/artists/cepp/contact.html [hunk 3]
```diff
@@ -35,7 +35,7 @@
                         <a href="contact.html">Contact</a>
                     </li>
                     <li>
-                        <a href="/">Sask Artists</a>
+                        <a href="../">Saskatchewan Artists</a>
```
Comment (gemohandy) [inline www/artists/cepp/contact.html]: ...This actually breaks the link. The original link worked fine. Check the actual website if you don't believe me.

Comment (Aholicknight) [reply www/artists/cepp/contact.html]: I will revert it to how it was before.

**Pull request merged**